mixer/test/client/network_policy/network_policy_test.go

```diff
@@ -78,7 +78,7 @@ func TestNetworkFailure(t *testing.T) {
 	if err != nil {
 		t.Errorf("Failed in request %s: %v", tag, err)
 	}
-	if code != 503 {
-		t.Errorf("Status code 503 is expected, got %d.", code)
+	if code == 200 {
+		t.Errorf("Non-200 status code 503 is expected, got %d.", code)
```

Comment:
curious: what changed re: status code in the update?

Also, you should remove "code 503" from the error message now.

Reply:
Thanks.  Removed it.

Envoy gRPC implementation has changed.   If gRPC server is not up,  it used to return status_code=LOSS_DATA.  Now it returns CACNCELED.